### PR TITLE
Add scalardl helm charts

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/browse/DLT-8785

# Done
- [scalar-kuberntes/charts/stable](https://github.com/scalar-labs/scalar-kubernetes/tree/master/charts/stable)
```
$  diff -r scalar-kuberntes/charts/stable helm-charts/charts
```
- [scalar-kubernetes/scripts](https://github.com/scalar-labs/scalar-kubernetes/tree/master/scripts)
```
diff -r scalar-kuberntes/scripts helm-charts/scripts
diff -r scalar-kuberntes/scripts/verify-chart-docs.sh helm-charts/scripts/verify-chart-docs.sh
7c7
< docs="${SCRIPT_ROOT}/../charts/stable/**/README.md"
---
> docs="${SCRIPT_ROOT}/../charts/**/README.md"
```
- [scalar-kuberntes/.github](https://github.com/scalar-labs/scalar-kubernetes/tree/master/.github)
```
diff -r scalar-kuberntes/.github helm-charts/.github                                                                                                             ✘ 1
diff -r scalar-kuberntes/.github/ct.yaml helm-charts/.github/ct.yaml
6c6
<   - charts/stable
---
>   - charts
diff -r scalar-kuberntes/.github/kubeval.sh helm-charts/.github/kubeval.sh
8c8
< CHART_DIRS="$(ls charts/stable)"
---
> CHART_DIRS="$(ls charts)"
24,25c24,25
<   echo "kubeval(idating) charts/stable/${CHART_DIR} chart..."
<   helm kubeval "charts/stable/${CHART_DIR}" -v "${KUBERNETES_VERSION#v}"
---
>   echo "kubeval(idating) charts/${CHART_DIR} chart..."
>   helm kubeval "charts/${CHART_DIR}" -v "${KUBERNETES_VERSION#v}"
diff -r scalar-kuberntes/.github/workflows/helm_charts_scalar.yml helm-charts/.github/workflows/helm_charts_scalar.yml
1a2
>
7c8
<       - "charts/stable/**"
---
>       - "charts/**"
12c13,14
<       - "charts/stable/**"
---
>       - "charts/**"
>
Only in scalar-kuberntes/.github/workflows: operation_ansible_lint.yml
Only in scalar-kuberntes/.github/workflows: operation_pluto_deprecated.yml
```

### CODE REVIEW

File: .github/workflows/helm_charts_scalar.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+name: Helm testing and validate for scalar charts
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (superbrothers) [inline .github/workflows/helm_charts_scalar.yml:6]:
```suggestion
      - main
```

File: .github/workflows/helm_charts_scalar.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    paths:
+      - "charts/**"
+  push:
+    branches:
+      - master
```
Comment (superbrothers) [inline .github/workflows/helm_charts_scalar.yml:11]:
```suggestion
      - main
```

**Review by ymorimo — APPROVED** [review]
LGTM (except for the branch names in the workflows)

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Comment by feeblefakie** [comment]
@tei-k One of the CI fails. Is it related to the fix?

**Comment by tei-k** [comment]
@feeblefakie 

>One of the CI fails. Is it related to the fix?

Fixed in 62284be.
https://github.com/helm/chart-testing-action/issues/25#issuecomment-687795744

File: .github/workflows/helm_charts_scalar.yml (hunk 1 above)
Comment (tei-k) [reply .github/workflows/helm_charts_scalar.yml]: Thanks! fixed in 8a6c79e.

File: .github/workflows/helm_charts_scalar.yml (hunk 2 above)
Comment (tei-k) [reply .github/workflows/helm_charts_scalar.yml]: Ditto.

**Comment by tei-k** [comment]
@feeblefakie Please add new secret env `CR_PAT`.

**Comment by superbrothers** [comment]
@tei-k @feeblefakie We can now use `GITHUB_TOKEN` to authenticate with `ghcr.io` in Actions workflows, so we no longer need `CR_PAT` here. Please see https://github.blog/changelog/2021-03-24-packages-container-registry-now-supports-github_token/ for more details.

**Comment by tei-k** [comment]
>We can now use GITHUB_TOKEN to authenticate with ghcr.io in Actions workflows, so we no longer need CR_PAT here

@superbrothers Thanks! I tryied it, but the ci fails, so I'll do it other PR.

@feeblefakie 
Please set `CR_PAT`

**Comment by superbrothers** [comment]
> Thanks! I tryied it, but the ci fails, so I'll do it other PR.

We need to change the permissions in the container settings as follows. This can be done in a separate PR.

> Write and read access of Actions to containers can be managed in the container settings.

https://github.blog/changelog/2021-03-24-packages-container-registry-now-supports-github_token/

**Comment by tei-k** [comment]
@feeblefakie 

Please confirm and set the following.
https://github.com/scalar-labs/helm-charts/pull/1#issuecomment-820877685
https://github.com/scalar-labs/helm-charts/pull/1#issuecomment-821917791

**Comment by feeblefakie** [comment]
@superbrothers Even though we changed the permission, it didn't work for some reason.
We will take another look at the issue later. For this PR, we use CR_PAT.
I merged it but PTAL again when you get a chance.

**Pull request merged**